### DISCUSSION TITLE
MSM: Add a way to hide sites from site switcher

### DISCUSSION THREAD

**Discussion by rowansentesy** [ExpressionEngine/ExpressionEngine#2949] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I was surprised to find out that I couldn't hide a site from the MSM switch_to menu. I presumed that by toggling the 'Website Online' option in the /cp/msm/edit screen that if set to 'off' the site would be hidden. A client of ours has old legacy sites that they don't want in the switcher, but aren't ready to delete them.

**Describe the solution you'd like**
Create a way to hide MSM sites from the switcher. Maybe the 'status' has 3 options: in addition to online & offline, add a 'hidden' or 'disabled' option. 

**Describe alternatives you've considered**
I've looked at using CSS and Javascript to hide these other sites, but this requires modifying the core EE theme files. 

**Teachability, Documentation, Adoption, Migration Strategy**
Manage your site's status by choosing the appropriate value from the Site Status dropdown menu.
Online - Your site is visible to the world
Offline - Your site is visible to super admins only, good for development purposes
Hidden - Your site is offline and will not be shown in the site switcher.

- [ Y ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
Investigated this, and if the member Role does not have CP access for a certain MSM site, it does not show up in the dropdown

So you would need a custom role with different level of access, should you need to hide some sites

Superadmins can see everything of course
